pipelines/measurement/measurement_tract.yaml

```diff
@@ -10,6 +10,9 @@ tasks:
         config.measure.retarget(TExTask)
         config.measure.minSep = 0.25
         config.measure.maxSep = 1.0
+        config.measure.column = "slot_Shape"
```

Comment:
It looks like the values being set here for `column` and `columnPsf` are the same as the defaults, so suggest that we don't specify them again in the yaml file.